haas/api.py

```diff
@@ -822,6 +813,9 @@ def show_node(nodename):
         'free': node.project_id is None,
         'nics': [{'label': n.label,
                   'macaddr': n.mac_addr,
+                  'networks': dict([(attachment.channel,
```

Comment:
I was thinking of something like https://stackoverflow.com/questions/9001509/how-can-i-sort-a-dictionary-by-key